doc/base/classes.xml

```diff
@@ -6052,10 +6055,14 @@
 	</methods>
 	<members>
 		<member name="atlas" type="Texture" setter="set_atlas" getter="get_atlas" brief="">
+			The texture that contains the atlas. Can be any Texture subtype.
 		</member>
 		<member name="margin" type="Rect2" setter="set_margin" getter="get_margin" brief="">
+			The margin around the region. 'x' and 'y' respectively define left and top gaps that margin adds to the resulting region. 'w' and 'h' respectively
+			define right and bottom margins. The two latter resize the texture so it fits in the defined margin.
```

Comment:
Favor explicit references over implicit ones. "latter" isn't a very common English word:
```The two latter resize the texture so it fits in the defined margin.```
-> also remove the passive voice:
```
The Rect2's 'w' and 'h' resize the texture so it fits within the margin.
```